go-controller/pkg/ovn/gateway_init.go

```diff
@@ -202,13 +231,25 @@ func gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet, hostSubnets []*n
 	// Create load balancers for workers (to be applied to GR and node switch)
 	for _, proto := range enabledProtos {
 		if workerProtoLBMap[proto] == "" {
-			workerProtoLBMap[proto], stderr, err = util.RunOVNNbctl("--", "create",
-				"load_balancer",
-				fmt.Sprintf("external_ids:%s-%s=%s", types.WorkerLBPrefix, strings.ToLower(string(proto)), nodeName),
-				fmt.Sprintf("protocol=%s", strings.ToLower(string(proto))))
-			if err != nil {
-				return fmt.Errorf("failed to create load balancer for worker node %s for protocol %s: "+
-					"stderr: %q, error: %v", nodeName, proto, stderr, err)
+			if config.Gateway.Mode != config.GatewayModeLocal {
+				workerProtoLBMap[proto], stderr, err = util.RunOVNNbctl("--", "create",
+					"load_balancer",
+					fmt.Sprintf("external_ids:%s-%s=%s", types.WorkerLBPrefix, strings.ToLower(string(proto)), nodeName),
+					fmt.Sprintf("protocol=%s", strings.ToLower(string(proto))),
+					fmt.Sprintf("options:lb_force_snat_ip=router_ip"))
```

Comment:
This change is not necessary right from lines 234-252? I'm guessing this is leftover from Lorenzo's first implementation?